Gemfile

```diff
@@ -7,6 +7,8 @@ gem 'dotenv'
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your branch name more descriptive such as `feature/hello-app-backend` instead of `feature` 👍 Your branch name should tell us what is happening in this pull request and make your pull request more descriptive and professional 😊

Reply:
Thanks for your suggestion, I'll keep it in mind in the future! 😄